rubin_sim/scheduler/model_observatory/model_observatory.py

```diff
@@ -215,11 +216,13 @@ def __init__(
         sun_moon_info = self.almanac.get_sun_moon_positions(self.mjd)
         season_offset = create_season_offset(self.nside, sun_moon_info["sun_RA"])
         self.sun_ra_start = sun_moon_info["sun_RA"] + 0
+        self.mjd_start = mjd_start
```

Comment:
Isn't this already done more reliably in line 121?

Reply:
Indeed!